emails/management/commands/process_delayed_emails_from_sqs.py

```diff
@@ -69,8 +69,12 @@ def handle(self, *args, **options):
         )
         while len(messages) > 0:
             for message in messages:
-                _verify_and_run_sns_inbound_on_message(message)
-
+                try:
+                    _verify_and_run_sns_inbound_on_message(message)
+                except:
+                    exc_type, _, _ = sys.exc_info()
+                    logger.exception(f'dlq_processing_error_{exc_type}')
```

Comment:
Will log the exception type, value, and traceback, see this [Sentry doc](https://docs.sentry.io/platforms/python/guides/logging/#usage) for more info.

Reply:
When we see these in Sentry, let's make sure these exceptions don't accidentally store any PII that we don't want to be storing.

Reply:
I thought about this too. Hopefully we can test it on Stage but if not I'll watch out for it the first time we run on Prod. But I believe this won't happen because we disabled the breadcrumbs which is where the PII shows up.